### DISCUSSION TITLE
"show displays as individual windows" is not work for me

### DISCUSSION THREAD

**Discussion by marktohark** [rustdesk/rustdesk#6676] [Q&A]
"show displays as individual windows" is not work for me
this is demo.

https://github.com/rustdesk/rustdesk/assets/19359934/dcabdfaf-0146-4841-8c37-8f8246f79cbd

![image](https://github.com/rustdesk/rustdesk/assets/19359934/e2c601d6-ade8-4b3f-9fb4-86b11ddc0924)

**Comment by fufesou** [comment]
What version is on the controlled side?

Only 1.2.4 support multiple screen instances.
So you should also use 1.2.4 on the controlled side.

<img width="317" alt="1702526674483" src="https://github.com/rustdesk/rustdesk/assets/13586388/5b27fd02-9b85-4cd7-ba15-08dc3a0b82a6">

**Reply by marktohark** [reply]
got it. it work.
